aca_lts_eval.py

```diff
@@ -255,7 +255,7 @@ def t_ccd_for_attitude(ra, dec, cycle, detector, too, y_offset=0, z_offset=0,
         raise ValueError("No stars found in 3 degree radius of {} {}".format(ra, dec))
 
     # get mag errs once for the field
-    cone_stars['mag_one_sig_err'], cone_stars['mag_one_sig_err2'] = mini_sausage.get_mag_errs(cone_stars)
+    #cone_stars['mag_one_sig_err'], cone_stars['mag_one_sig_err2'] = mini_sausage.get_mag_errs(cone_stars)
```

Comment:
I think this can safely be uncommented (the mag errors are not specific to acquisition or guide selection).